### PR TITLE
What are job containers about?

### PR DESCRIPTION
**Pull request by chadwhitacre**
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idcontainer

### CODE REVIEW

**Comment by chadwhitacre** [comment]
@BYK I think this is going to be the better approach, [`container`](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idcontainer) instead of [`uses`](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsuses) (cf. #4), because it means we can have fine-grained output in the Actions UI vs. burying everything underneath a single "Install and test" step (we'd share config across envs using YAML merges rather than via shared shell scripts). I've got the basic shape of the thing done in 6f059a6526255b55d7b584f89119bf96d740f866, but `install.sh` is failing with an [error](https://github.com/chadwhitacre/onpremise/pull/5/checks?check_run_id=1031360426#step:7:557) that's not immediately obvious to me:

> ERROR: for symbolicator  Cannot create container for service symbolicator: invalid mount config for type "bind": bind source path does not exist: /__w/onpremise/onpremise/symbolicator

I've started debugging, familiarizing myself with the `docker-compose.yml`. I'll pick up there next chance I get.

**Comment by chadwhitacre** [comment]
https://stackoverflow.com/questions/60910436/docker-compose-bind-mount-doesnt-work-in-github-actions

**Comment by chadwhitacre** [comment]
Dang so I think the context for the bind mount is the host filesystem, not the container filesystem. 🤔

**Comment by chadwhitacre** [comment]
Gosh yeah that's gnarly. That means we need to ...

https://stackoverflow.com/questions/55845723/volume-bind-mounting-from-docker-in-docker

**Comment by chadwhitacre** [comment]
tl;dr not use a bind mount, which means changing docker-compose.yml. I hate to see changes driven by limitations of the CI infrastructure. Is there a way we can mirror the filesystem on the host in GHA? 🤔
